bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -6,7 +6,9 @@
 
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}
-{% if l10n_has_tag('tracking_protection') %}
+{% if l10n_has_tag('fx49_remove_hello') %}
+  {{ _('Firefox – Desktop browser, Android, iOS, OS, Sync, Private Browsing') }}
```

Comment:
Should we drop "OS" from this string as well? (Not sure if L10n already started here...)